### PR TITLE
edit tokenizer in function calling tutorial

### PR DESCRIPTION
**Pull request by Erika-Russi**

### CODE REVIEW

File: generative-ai/function-calling.ipynb
Comment (AnnaGutowska) [inline generative-ai/function-calling.ipynb:1]: Please change the line above to "and the tokenizer for the same Granite model."

**Review by AnnaGutowska — CHANGES_REQUESTED** [review]
Please change the line above to "and the tokenizer for the same Granite model."

File: generative-ai/function-calling.ipynb
Comment (Erika-Russi) [reply generative-ai/function-calling.ipynb:1]: good catch!

**Comment by Erika-Russi** [comment]
> Please change the line above to "and the tokenizer for the same Granite model."

done thanks!

File: generative-ai/function-calling.ipynb
Comment (AnnaGutowska) [reply generative-ai/function-calling.ipynb:1]: Also, @Erika-Russi please delete the reference in line 15 to the "Granite-20B-FunctionCalling tokenizer"

**Review by Jess-Bo — APPROVED** [review]
LGTM.

**Pull request merged**